dhis-2/dhis-services/dhis-service-tracker/src/test/java/org/hisp/dhis/tracker/export/trackedentity/TrackedEntityOperationParamsMapperTest.java

```diff
@@ -638,6 +639,55 @@ void shouldMapTeaWhenTeaMinCharactersSetAndReached()
         List.of(TEA_1_UID.getValue()), UID.toUidValueSet(queryParams.getFilters().keySet()));
   }
 
+  @Test
+  void shouldMapAttributeFiltersWhenOperatorsAreNotBlocked()
+      throws ForbiddenException, BadRequestException {
+    when(attributeService.getTrackedEntityAttribute(TEA_2_UID.getValue())).thenReturn(tea2);
+    when(aclService.canDataRead(any(UserDetails.class), any(TrackedEntityType.class)))
+        .thenReturn(true);
+    when(paramsValidator.validateTrackedEntityType(UID.of(trackedEntityType), user))
+        .thenReturn(trackedEntityType);
+    when(trackedEntityTypeService.getAllTrackedEntityType()).thenReturn(List.of(trackedEntityType));
+
+    TrackedEntityOperationParams trackedEntityOperationParams =
+        TrackedEntityOperationParams.builder()
+            .trackedEntityType(trackedEntityType)
+            .filterBy(
+                TEA_2_UID,
+                List.of(
+                    new QueryFilter(LIKE, "12"),
+                    new QueryFilter(NNULL),
+                    new QueryFilter(ILIKE, "0")))
+            .build();
+
+    TrackedEntityQueryParams queryParams = mapper.map(trackedEntityOperationParams, user);
+    assertContainsOnly(
+        List.of(TEA_2_UID.getValue()), UID.toUidValueSet(queryParams.getFilters().keySet()));
+  }
+
+  @Test
+  void shouldNotMapAttributeFiltersWhenOperatorsAreBlocked()
```

Comment:
```suggestion
  void shouldFailToMapAttributeFiltersWhenOperatorsAreBlocked()
```
so it is clear that an exception is thrown.